mmv1/third_party/terraform/services/orgpolicy/resource_org_policy_policy_test.go

```diff
@@ -458,3 +458,31 @@ func testAccCheckOrgPolicyPolicyDestroyProducer(t *testing.T) func(s *terraform.
 		return nil
 	}
 }
+
+func testAccOrgPolicyPolicy_EnforceParameterizedMCPolicy(context map[string]interface{}) string {
```

Comment:
this test configuration needs to be referenced in a test runner function (capital "TestAccOrgPolicy...")

Also, can we get a configuration for the dryrun spec variant as well?